### PR TITLE
Add ReportTab

### PR DESCRIPTION
**Pull request by huynhngocminh2511**

### CODE REVIEW

File: Zen/Activities/HomeListenActivity.cs [hunk 1]
```diff
@@ -79,11 +79,13 @@ private void InitSounds()
 
             configuration = UserConfiguration.Instance.Configuration;
 
-            if (configuration.BellEnable && soundsController.BellSelected.Url != null) PlaySoundIntervalTime(soundsController.BellSelected);
+            //if (configuration.BellEnable && soundsController.BellSelected.Url != null)
```
Comment (voquanghoa) [inline Zen/Activities/HomeListenActivity.cs]: Remove dead code

File: Zen/Controllers/SoundsController.cs [hunk 2]
```diff
@@ -26,7 +18,7 @@ class SoundsController
 
         private const string ambientSound = "ambient";
 
-        private Configuration.Configuration configuration = UserConfiguration.Instance.Configuration;
+        //private Configuration.Configuration configuration = UserConfiguration.Instance.Configuration;
```
Comment (voquanghoa) [inline Zen/Controllers/SoundsController.cs]: Remove dead code

File: Zen/Fragments/SettingFragment.cs [hunk 3]
```diff
@@ -65,6 +66,9 @@ public override View OnCreateView(LayoutInflater inflater, ViewGroup container,
         {
             var view = inflater.Inflate(Resource.Layout.SettingLayout, container, false);
             Cheeseknife.Inject(this, view);
+            var e = UserConfiguration.Instance;
```
Comment (voquanghoa) [inline Zen/Fragments/SettingFragment.cs]: Useful code?

File: Zen/Activities/HomeListenActivity.cs (hunk 1 above)
Comment (huynhngocminh2511) [reply Zen/Activities/HomeListenActivity.cs]: Done

File: Zen/Controllers/SoundsController.cs (hunk 2 above)
Comment (huynhngocminh2511) [reply Zen/Controllers/SoundsController.cs]: Done

File: Zen/Fragments/SettingFragment.cs (hunk 3 above)
Comment (huynhngocminh2511) [reply Zen/Fragments/SettingFragment.cs]: Done

**Pull request merged**